src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -7,14 +7,42 @@
 
 package frc.robot.subsystems;
 
+import edu.wpi.first.wpilibj.VictorSP;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import frc.robot.Constants;
 
 public class IntakeSubsystem extends SubsystemBase {
-  /**
-   * Creates a new ExampleSubsystem.
-   */
+  private final VictorSP intakeSPrt = new VictorSP(Constants.INTAKE_VICTORSP_PWM_CHAN_LF);
+  private final VictorSP intakeSPlf = new VictorSP(Constants.INTAKE_VICTORSP_PWM_CHAN_RT);
+
+  private final DoubleSolenoid intakePistonL = new DoubleSolenoid(Constants.INTAKE_PISTON_PCM_CHAN_LF_F,
+      Constants.INTAKE_PISTON_PCM_CHAN_LF_R);
+  private final DoubleSolenoid intakePistonR = new DoubleSolenoid(Constants.INTAKE_PISTON_PCM_CHAN_RT_F,
+      Constants.INTAKE_PISTON_PCM_CHAN_RT_R);
+
   public IntakeSubsystem() {
+    intakeSPrt.setInverted(true);
+  }
+
+  public void runIntake() {
+    intakeSPrt.set(0.3);
+    intakeSPlf.set(0.3);
+  }
+
+  public void stopIntake() {
+    intakeSPrt.stopMotor();
+    intakeSPlf.stopMotor();
+  }
+
+  public void raiseIntake() {
+    intakePistonL.set(DoubleSolenoid.Value.kForward);
```

Comment:
Should replace reference to kForward etc with class constants to abstract Deploy vs Stow.